README.md

```diff
@@ -387,7 +387,7 @@ Once installing the plug-in the usage is similar to the built in checkers.
 1. Run the NCEI Point check on a THREDDS endpoint
 
 ```python
-compliance-check -t ncei-point -v https://data.nodc.noaa.gov/thredds/dodsC/testdata/mbiddle/GOLD_STANDARD_NETCDF/1.1/NODC_point_template_v1.1_2016-06-14_125317.379316.nc
+compliance-checker -t ncei-point -v "https://data.nodc.noaa.gov/thredds/dodsC/testdata/mbiddle/GOLD_STANDARD_NETCDF/1.1/NODC_point_template_v1.1_2016-06-15_133710.844375.nc"
```

Comment:
The `cc-plugin-ncei` is needed to run this one, right?

Reply:
Yeah this example is in the Compliance Checker Plug-Ins section

Reply:
Cool. LGTM!